js_modules/dagster-ui/packages/ui-core/src/asset-data/__tests__/AssetLiveDataProvider.test.tsx

```diff
@@ -380,9 +377,6 @@ describe('AssetLiveDataProvider', () => {
     });
 
     expect(resultFn).toHaveBeenCalled();
-    await waitFor(() => {
-      expect(hookResult.mock.calls[1]!.value).not.toEqual({});
```

Comment:
this was an unnecessary render that was removed. Since we error'd there's no reason to re-render the component.